packages/kit/src/runtime/client/router.js

```diff
@@ -8,11 +8,11 @@ function scroll_state() {
 }
 
 /**
- * @param {Node | null} node
- * @returns {HTMLAnchorElement | SVGAElement | null}
+ * @param {Node[]} nodes
+ * @returns {HTMLAnchorElement | SVGAElement | undefined}
  */
-function find_anchor(node) {
-	while (node && node.nodeName.toUpperCase() !== 'A') node = node.parentNode; // SVG <a> elements have a lowercase name
+function find_anchor(nodes) {
```

Comment:
I think we should be receiving the event instead and do `event.composedPath()` inside this function to organize the its usage.

Reply:
Done